### DISCUSSION TITLE
Enhancing First Issues Tagging for Beginner-Friendly Contributions

### DISCUSSION THREAD

**Discussion by ANcpLua** [hari7261/Hacktoberfest-2024#49] [Ideas]
I’m super excited about Hacktoberfest this year! I’ve already contributed to a couple of repositories, but I noticed many beginners struggle with understanding good first issues. To make it easier, could we introduce a tagging system that provides clearer context on the difficulty level of an issue? For example, tags like 'setup knowledge required,' 'API experience,' or 'UI-focused' could help participants gauge where their skills fit best. This would likely increase engagement and improve the quality of contributions. Thoughts on this idea?

**Comment by hari7261** [comment]
Hey @ANcpLua ,
Awesome idea! Adding tags like ‘setup knowledge,’ ‘API experience,’ or ‘UI-focused’ would make it way easier for beginners to find issues that match their skills. It'd boost engagement and help maintainers get more targeted contributions. Definitely a win for Hacktoberfest!
